db/data_migrate/20190408132216_fix_american_dates.rb

```diff
@@ -0,0 +1,41 @@
+# The CSV file was generated with the help of db/data_migrate/correct_american_data_formats.rb
+#
+# Execute with:
+#
+#   rails runner db/data_migrate/20190408132216_fix_american_dates.csv
```

Comment:
I don't think we want to be running the rails runner against a csv!

Reply:
🤦‍♂️